hivemind/client/averaging/matchmaking.py

```diff
@@ -165,7 +166,7 @@ async def request_join_group(self, leader: Endpoint, expiration_time: DHTExpirat
                 leader_stub = ChannelCache.get_stub(leader, averaging_pb2_grpc.DecentralizedAveragingStub, aio=True)
                 call = leader_stub.rpc_join_group(averaging_pb2.JoinRequest(
                     endpoint=self.endpoint, schema_hash=self.schema_hash, expiration=expiration_time,
-                    throughput=self.throughput if self.throughput is not None else -1.0,
+                    throughput=0.0 if self.client_mode else (self.throughput if self.throughput is not None else -1.0),
```

Comment:
The triple if condition seems too complex (especially for a single line), can't we break this down or reduce the number of cases?